auto_round/autoround.py

```diff
@@ -1611,11 +1620,12 @@ def calib(self, nsamples, bs):
             for n, m in embed_layers:
                 m = m.to(self.device)
 
+        device = self.model.device if device is None else device
         for data in self.dataloader:
             if data is None:
                 continue
             if isinstance(data, torch.Tensor):
-                input_ids = data.to(self.model.device)
+                input_ids = data.to(device)
```

Comment:
why changing this code, for calibration,  if cuda oom, the model will be transferred to cpu, then the change will cause bugs

Reply:
Could you please help add a CUDA test for a 70B model? This will help ensure any bugs related to this issue are caught early.